packages/flutter_tools/lib/src/commands/listen.dart

```diff
@@ -11,8 +11,8 @@ import 'start.dart';
 
 class ListenCommand extends StartCommandBase {
   final String name = 'listen';
-  final String description = 'Listen for changes to files and reload the running app on all connected devices. (Android only.)'
-      'By default, only listens to "./" and "./lib/". To listen to additional directories, list them on the command line.';
+  final String description = 'Listen for changes to files and reload the running app on all connected devices (Android only).'
+      ' By default, only listens to "./" and "./lib/". To listen to additional directories, list them on the command line.';
```

Comment:
please make sure logs, trace, listen, and start are all consistent in how this is punctuated.